src/Countdown.elm

```diff
@@ -43,8 +42,13 @@ kickOff =
     Time.millisToPosix 1578154400000
 
 
-subscriptions : Model -> Sub Msg
-subscriptions _ =
+addStrings : String -> String -> String -> String -> String -> String
+addStrings week day hour minute second =
+    week ++ " weeks" ++ " , " ++ day ++ " Days" ++ " , " ++ hour ++ " Hours" ++ " , " ++ minute ++ " Minutes" ++ " and " ++ second ++ " seconds"
```

Comment:
This is very error-prone.
One may easily confuse the different parameters.
When we need so many parameters, we usually just pass a record.
Also, you repeat `String.fromInt` (on line 102) and `++` (here) a lot.
How about:
```elm
addStrings : { week : Int, day : Int, hour : Int, minute : Int, second : Int } -> String
addStrings { week, day, hour, minute, second } =
    List.concat
        [ String.fromInt week,
        , " weeks, "
        , String.fromInt day
        , " days, "
        , String.fromInt hour
        , " hours, "
        , String.fromInt minute
        , " minutes and "
        , String.fromInt second
        , " seconds"
        ]
```
which is used like this:
```elm
addStrings { minute = 5, week = 1, day = 3, hour = 2, second = 23 } -- The order of the values inside the record is meaningless
```